src/core/function_subscription_migrations.tf

```diff
@@ -0,0 +1,87 @@
+locals {
+  app_settings_commons = {
+    FUNCTIONS_WORKER_RUNTIME       = "node"
+    WEBSITE_NODE_DEFAULT_VERSION   = "14.16.0"
+    FUNCTIONS_WORKER_PROCESS_COUNT = 4
+    NODE_ENV                       = "production"
+
+    // Keepalive fields are all optionals
+    FETCH_KEEPALIVE_ENABLED             = "true"
+    FETCH_KEEPALIVE_SOCKET_ACTIVE_TTL   = "110000"
+    FETCH_KEEPALIVE_MAX_SOCKETS         = "40"
+    FETCH_KEEPALIVE_MAX_FREE_SOCKETS    = "10"
+    FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
+    FETCH_KEEPALIVE_TIMEOUT             = "60000"
+  }
+
+  // As we run this application under SelfCare IO logic subdomain, 
+  //  we share some resources
+  app_context = {
+    resource_group   = azurerm_resource_group.selfcare_be_rg
+    app_service_plan = azurerm_app_service_plan.selfcare_be_common
+    snet             = module.selfcare_be_common_snet
+  }
```

Comment:
locals are global so it's better to define inner object, for example:

```
locals {

 function_subscriptionmigrations = {
  app_context = {
  }
  app_settings_commons = {
  }
 } 


}
```

Reply:
isn't there a file-scoped way to declare variables?

Reply:
no :( terraform reads all files in a folder